### ISSUE TITLE
Nothing found by Chrome extension (Documentation Issue...)

### ISSUE THREAD

**Issue by LorenAmelang** [lovasoa/dezoomify-extension#22]
I'm new to all of this...  Would like to save bits of
https://artsandculture.google.com/exhibit/ruth-asawa-a-working-life-de-young-museum/hAIygGBPp11bIg?hl=en&mc_cid=4d0a8bec9b&mc_eid=1fbadca0ed

Tried 
https://ophir.alwaysdata.net/dezoomify/dezoomify.html
It found nothing. 

Got the Chrome extension, in 
Vivaldi | 3.6.2165.40 (Stable channel) (64-bit)
OS | Windows 10 OS Version 2004 (Build 19041.867)

(Also tried it in Firefox, same behavior.)

Yes, I tried disabling uBlock and CleanURLs, didn't help. 

Page reloads, icon turns blue, I zoom into images by double-clicking them, or into maps with the +/- icons, no badge appears, mouseover still says it is listening. 

Somehow ended up on
https://artsandculture.google.com/asset/stem-with-leaves-ruth-asawa/iQHugBHl-G9Cow?hl=en&ms=%7B%22x%22:0.5,%22y%22:0.5316622691292876,%22z%22:9.907584514601407,%22size%22:%7B%22width%22:1.4027453824689835,%22height%22:0.6000000000000001%7D%7D
It zooms even more, but still no badge, nothing found. ...  Oh - I got it to max zoom, couldn't figure out how to unzoom. Randomly tried double-right-clicking it, and got a badge! And it went to the dezoomify web page and saved it! 

Can't make that work again, double-right-clicking now gets a browser menu. The single image web pages zoom with the mouse wheel, and have an obvious '+' button. And they get a dezoomify badge as soon as you zoom them! 

Guess my main problem was being on the first URL multiple image page, with the mouse wheel zoom scrolling to the right and tiny diamond icons along the bottom. You can make those images sort-of zoom, but you can't save them! 

Still not sure how I got from the first overall URL to the single image URL. Or how to get back to the multi image page. 
Oh - Sometimes you can use the browser "back" button to get back to the first URL - but sometimes not...  
And you can click the white image title to move to the single page view. But there must be other ways, I didn't do that the first time...  

Probably irrelevant, but...  
Checked DevTools, there are 63 of these: 

[Violation] Added non-passive event listener to a scroll-blocking <some> event. Consider marking event handler as 'passive' to make the page more responsive. See <URL>
/culturalinstitute/_/scs/ci-static/_/js/k=culturalinstitute-m.stella.en.Lh4jkcnDJy4.O/am=IABg/d=1/excm=IPbtr,__base/ed=1/ct=zgms/rs=AIxt-LqUgE7grGBzJ83lTIIrc69Cu5SSBA/m=stella:200 

[Violation] Added non-passive event listener to a scroll-blocking 'touchmove' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952
zh @ /culturalinstitute/_/scs/ci-static/_/js/k=culturalinstitute-m.stella.en.Lh4jkcnDJy4.O/am=IABg/d=1/excm=IPbtr,__base/ed=1/ct=zgms/rs=AIxt-LqUgE7grGBzJ83lTIIrc69Cu5SSBA/m=stella:200

(With different "stella:" numbers...)

Peeked at Network:XHR, there are 112 requests, 1.5 MB, no idea which might be the file you need...  

Guess I've figured this out, but maybe it will help some other newbie...

**Comment by lovasoa** [comment]
Yes, you have to be on an individual asset page for dezoomify to work. From the exhibit page, you just click the asset name:

<img width="204" alt="image" src="https://user-images.githubusercontent.com/552629/112811586-82d0d280-907c-11eb-831e-48c07557b57d.png">


Once you are on the asset page, there is nothing to do, the extension automatically finds the image:

![Uploading image.png…]()

**Comment by LorenAmelang** [comment]
Could you _Please_ add your above clue to the "How to Use" section about your extension? "You have to be on an individual asset page for dezoomify to work. From the exhibit page, you just click the asset name" could have saved me _so_ much frustration! 

As far as I could tell the "exhibit page" images were zoomable...

**Comment by lovasoa** [comment]
Ideally, the extension should even be made to work on exhibition pages.
In the meantime, you can contribute to the dezoomify wiki: https://github.com/lovasoa/dezoomify/wiki
